tests/integration/cli/repos_test.py

```diff
@@ -0,0 +1,69 @@
+from os import path
+import shutil
+from subprocess import check_output
+
+import git
+
+from dusty.constants import REPOS_DIR
+from ...fixtures import busybox_single_app_bundle_fixture
+from ...testcases import DustyIntegrationTestCase
+
+
+class TestReposCLI(DustyIntegrationTestCase):
+    def setUp(self):
+        super(TestReposCLI, self).setUp()
+        busybox_single_app_bundle_fixture(num_bundles=1)
+        self.set_up_fake_local_repo(path='/tmp/fake-repo-1', short_name='fake-repo-1')
```

Comment:
Should be using `tempfile.mkdtemp` instead of this, I don't think it's always valid to assume `/tmp` is a writable location.